pkg/debugapi/postage.go

```diff
@@ -66,6 +66,14 @@ func (s *Service) postageCreateHandler(w http.ResponseWriter, r *http.Request) {
 		immutable, _ = strconv.ParseBool(val[0])
 	}
 
+	if !s.postageCreateSem.TryAcquire(1) {
+		s.logger.Debug("create batch: resource is busy")
```

Comment:
I would add an Error here too, for the user to see even if running not in debug log level